common-lib/src/main/scala/com/gu/mediaservice/model/usage/UsageStatus.scala

```diff
@@ -7,13 +7,16 @@ trait UsageStatus {
   override def toString = this match {
     case _:PendingUsageStatus => "pending"
     case _:PublishedUsageStatus => "published"
+    case _:FrontUsageStatus => "front_usage"
```

Comment:
Not sure this is makes sense, semantically. We almost need a `platform` or similar in the model.

Reply:
I understand what you mean, this doesn't really make sense but since we don't distinguish between pending and published usages in fronts usages it sort of makes sense, the status is not pending and it's not published because it is a different type of usage.

Reply:
So the search chip would be `usages@status:fronts` like `usages@status:published/pending`?
 as opposed to `usages@platform:print/digital`? Hmm… a little confusing indeed, but as Reetta said front overrides do not really have a `pending`/`published` status as such…